e2e/support/helpers/e2e-viz-settings-helpers.js

```diff
@@ -13,3 +13,10 @@ export function openSeriesSettings(field, isBreakout = false) {
       .click();
   }
 }
+
+export function openVizType(tabName = "Chart") {
```

Comment:
nit: on mobile the footer with "visualize" button should not be scrollable horizontally - only the query area should

![image](https://github.com/user-attachments/assets/e26d8141-57a8-4e38-b033-c261b6feb051)

Reply:
I'm not sure about it, it's more for @AlessioLaiso @mngr

Reply:
Yes, the footer should stay in place, left-aligned.